### DISCUSSION TITLE
getting unwanted entire seasons

### DISCUSSION THREAD

**Discussion by mark-hahn** [Flexget/Flexget#3103] [Q&A]
My flexget is downloading entire seasons of shows and filling up my disk.  I only want episodes.  Can I stop this?  For example, I just got 6 seasons of Drew Carey which was 45 GB (ouch).

**Comment by ksurl** [comment]
you can set `season_packs: no`

https://flexget.com/Plugins/series/season_packs

**Comment by liiight** [comment]
By default, season packs are turned off. Did you enable it in your config?

**Reply by mark-hahn** [reply]
I did not have season packs enabled and I was definitely getting them.  I'll add `season_packs: no` just in case.  If it happens again I'll post a bug, but I have other countermeasures in place so it may not happen.
